### ISSUE TITLE
'o3d.visualization.draw_geometries' is not working when I pass camera parameters to it.

### ISSUE THREAD

**Issue by hongniaoNO1** [isl-org/Open3D#6858]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](https://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](https://www.open3d.org/docs/release/) and the [latest documentation](https://www.open3d.org/docs/latest/) (for `main` branch).

### Describe the issue

'o3d.visualization.draw_geometries' is not working when I pass camera parameters to it.Terminal display’Process finished with exit code -1073741819 (0xC0000005)‘.but I removed the camera parameters, and the image was produced normally

### Steps to reproduce the bug

```python
import open3d as o3d
print("Load a ply point cloud, print it, and render it")
sample_pcd_data = o3d.data.PCDPointCloud()
pcd = o3d.io.read_point_cloud(sample_pcd_data.path)
o3d.visualization.draw_geometries([pcd],
                                  zoom=0.3412,
                                  front=[0.4257, -0.2125, -0.8795],
                                  lookat=[2.6172, 2.0475, 1.532],
                                  up=[-0.0694, -0.9768, 0.2024])
```

### Error message

Terminal display'Process finished with exit code -1073741819 (0xC0000005)'

### Open3D, Python and System information

```markdown
- Operating system:  Windows 11 64-bit
- Python version: Python 3.10.11 / output from `import sys; print(sys.version)`
- Open3D version: output from python: `print(open3d.__version__)`0.18.0
- System architecture: x64
- Is this a remote workstation?: no
- How did you install Open3D?: pip 
- Compiler version (if built from source): gcc 7.5 / clang 7.0
```

**Comment by baimuwu888** [comment]
I encountered the same issue, and it was resolved after switching to Python 3.8.

**Comment by hongniaoNO1** [comment]
Thank you, the problem is solved

> I encountered the same issue, and it was resolved after switching to Python 3.8.我遇到了同样的问题，切换到 Python 3.8 后得到了解决。

**Comment by hahakid** [comment]
replace numpy=2.X with numpy==1.24.0
